lib/features.js

```diff
@@ -14,6 +14,16 @@
 // Public
 //------------------------------------------------------------------------------
 
+/**
+ * @typedef {Object} EcmaFeatures
+ * @property {boolean} jsx
+ * @property {boolean} globalReturn
+ * @property {boolean} impliedStrict
+ */
```

Comment:
```suggestion
/**
 * @typedef {Object} EcmaFeatures
 * @property {boolean} [jsx]
 * @property {boolean} [globalReturn]
 * @property {boolean} [impliedStrict]
 */
```

Same as the previous comment.